src/events/preventEmojiSpam.ts

```diff
@@ -0,0 +1,23 @@
+import { Events, Message } from 'discord.js'
```

Comment:
Would be nice if you delete this file. It is already there in your previous PR. So we can discuss the commands separately.